lib/merchant_repository.rb

```diff
@@ -1,18 +1,20 @@
-require 'csv'
 require 'bigdecimal'
 require 'bigdecimal/util'
 require_relative 'merchant'
+require_relative 'file_loader'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.